specification/purview/data-plane/Azure.Analytics.Purview.DataPolicies/2022-02-01-preview/purviewDevopsPolicy.json

```diff
@@ -0,0 +1,397 @@
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "The error code"
+              }
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink",
+          "itemName": "values"
+        },
+        "x-ms-examples": {
+          "DevopsPolicy_List": {
+            "$ref": "./examples/DevopsPolicy_List.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "DataFabricPolicyContentDetails": {
+      "type": "object",
+      "properties": {
+        "policyType": {
+          "type": "string"
+        },
+        "identificationHash": {
+          "type": "string"
+        }
+      }
+    },
+    "ErrorModel": {
+      "description": "The error model for policy",
+      "required": [
+        "code",
+        "message"
+      ],
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "The error code",
+          "minLength": 1,
+          "type": "string"
+        },
+        "message": {
+          "description": "The error message",
+          "minLength": 1,
+          "type": "string"
+        },
+        "target": {
+          "description": "The error target",
+          "type": "string"
+        },
+        "details": {
+          "description": "The error details",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ErrorModel"
+          }
+        }
+      }
+    },
+    "ErrorResponseModel": {
+      "description": "The error response model for policy",
+      "required": [
+        "error"
+      ],
+      "type": "object",
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/ErrorModel"
+        }
+      }
+    },
+    "DevopsPolicy": {
+      "required": [
+        "decisionRules",
+        "scope"
+      ],
+      "type": "object",
+      "properties": {
+        "policyName": {
+          "type": "string"
+        },
+        "id": {
+          "type": "string"
+        },
+        "systemData": {
+          "$ref": "#/definitions/SystemData"
+        },
+        "policyDetails": {
+          "$ref": "#/definitions/DataFabricPolicyContentDetails"
+        },
+        "version": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "decisionRules": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DevopsPolicyDecisionRule"
+          }
+        },
+        "scope": {
+          "$ref": "#/definitions/DevopsPolicyScope"
+        }
+      }
+    },
+    "DevopsPolicyList": {
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DevopsPolicy"
+          }
+        },
+        "nextLink": {
+          "type": "string"
+        }
+      }
+    },
+    "DevopsPolicyDecisionRule": {
```

Comment:
All the properties in this schema also need descriptions. Users will have no idea what information should be specified in these fields without descriptions. See [the preview docs](https://review.docs.microsoft.com/en-us/rest/api/documentation-preview/devops-policy/create-or-update?view=azure-rest-preview&branch=result_d80254f2-1144-4bcc-9460-41a032ba187e&tabs=HTTP#devopspolicydecisionrule).